frontend/test/__support__/ui.tsx

```diff
@@ -71,6 +72,53 @@ export function renderWithProviders(
     ...options
   }: RenderWithProvidersOptions = {},
 ) {
+  const { wrapper, store, history } = getTestStoreAndWrapper({
+    mode,
+    initialRoute,
+    storeInitialState,
+    withRouter,
+    withKBar,
+    withDND,
+    withUndos,
+    customReducers,
+    theme,
+    ...options,
+  });
+
+  const utils = testingLibraryRender(ui, {
+    wrapper,
+    ...options,
+  });
+
+  return {
+    ...utils,
+    store,
+    history,
+  };
+}
+
+export function renderHookWithProviders<TProps, TResult>(
+  hook: (props: TProps) => TResult,
+  options?: Omit<RenderHookOptions<TProps>, "wrapper"> &
+    RenderWithProvidersOptions,
+) {
+  const { wrapper, store } = getTestStoreAndWrapper(options);
+  const renderHookReturn = renderHook(hook, { wrapper, ...options });
```

Comment:
this feels a bit awkward. We pass the same `options` to both `getTestStoreAndWrapper` and `renderHook` and both mustn't have the same type.

Can we be more explicit with what option is for what? Like maybe this function accept 3 arguments `hook`, our `options` and `renderHookOptions` or something.